### PR TITLE
Add tests demonstrating problem with duplicate rebuses

### PR DESCRIPTION
**Pull request by samanpwbb**
See tests. Either we figure out how to gracefully handle this case in the XD, or we throw an error and make the .puz to .xd parser handle it by never creating duplicate rebuses.

### CODE REVIEW

File: tests/xdToJSON.test.ts [hunk 1]
```diff
@@ -484,3 +484,61 @@ A1. Example ~ TWITCH|DOT|T|V
...
+A6. to live ~ EAT
+
+D1. the letter r ~ AR
+D2. skin ink ~ TAT
+D3. nice animal ~ CAT
+D4. two vowels ~ OE`
+
+  const json = xdToJSON(puzzle)
+  const newXD = JSONToXD(json)
+  expect(newXD).toEqual(puzzle)
```
Comment (samanpwbb) [inline tests/xdToJSON.test.ts:543]: this fails:
```
      ## Clues

      A1. to ~ AT
      A3. company ~ CO
    - A5. rodent ~ RAT
    + A5. rodent ~ RAT ❷
    - A6. to live ~ EAT
    + A6. to live ~ ❷T

      D1. the letter r ~ AR
    - D2. skin ink ~ TAT
    + D2. skin ink ~ TAT ❷
    - D3. nice animal ~ CAT
    + D3. nice animal ~ C❷
      D4. two vowels ~ OE
```

**Review by cod1r — APPROVED** [review]

**Comment by orta** [comment]
Yeah, this all makes sense to me, nice digging
